Submitting a New Hybrid Genome Assembly Tutorial

tflowers15 opened the pull request:
I am submitting a new tutorial on performing hybrid genome assembly using Galaxy. This tutorial is based on a tutorial created by Melbourne Bioinformatics.

This pull request includes the tutorial markdown file and tutorial images and an associated workflow and workflow tests. The CONTRIBUTORS.yaml file has also been updated to include new details for one of the co-authors of the tutorial. All datasets used in this tutorial are available in a published zenodo repository.

I confirm that all images are allowed to be re-hosted by the GTN.

hexylena commented on topics/assembly/tutorials/hybrid_denovo_assembly/tutorial.md:
```diff
@@ -0,0 +1,626 @@
+> {: .solution}
+> 
+{: .question}
+
```
Would including bandage here be instructive for the learners?

tflowers15 replied:
I did consider including a step on running bandage to view the assembly graph, however I don't think it provides much insight. I would prefer to not include it when I run local workshops using this tutorial. This tutorial is based on the workshops that I run at The University of Melbourne.

Delphine-L reviewed (approved):
Thank you for this tutorial, it looks good! I just have a couple of suggestions to help with the learner understanding.

tflowers15 commented on topics/assembly/tutorials/hybrid_denovo_assembly/tutorial.md:
```diff
@@ -0,0 +1,626 @@
+
+Mixtures of bacterial types can be sequenced e.g. prepare genomic DNA from environmental samples containing bacteria - water, soil, faecal samples etc. (Whole Metagenome Sequencing)
+
+![image](https://user-images.githubusercontent.com/10491057/93856582-8d409700-fcfc-11ea-9f0c-951e4ab05519.png)
```
I have uploaded the image to the images folder and replaced the path in the tutorial.

tflowers15 commented on topics/assembly/tutorials/hybrid_denovo_assembly/tutorial.md:
```diff
@@ -0,0 +1,626 @@
+> 
+> > <solution-title></solution-title>
+> > 
+> > We see that many genes are fragmented or missing. Our draft genome assembly isn't as good as the reference genome ***yet***.
```
I have expanded the solution to this question and now refer to the higher base-level error rate in long-read data compared to short-read data and that the next steps will use short-read data to correct for these errors.

tflowers15 commented on topics/assembly/tutorials/hybrid_denovo_assembly/tutorial.md:
```diff
@@ -0,0 +1,626 @@
+
+To do this, the best course of action would be to generate more long-read data.
+
+
```
Thank you for the suggestion. This is a great inclusion as it makes it very easy to compare the quality metrics for the genome assemblies. I have updated the tutorial to add a step for running multiqc and update the quast and busco instructions accordingly (output quast tabular reports and rename quast and multiqc outputs to make it easier to select them when running multiqc). I have also added images from the multiqc webpage output to the images directory.

tflowers15 commented:
@hexylena and @Delphine-L Thank you both for your comments and edits. I have addressed all of them and either accepted the proposed commit, updated the tutorial as suggested or replied to your comment when I have not fully implemented the suggested change (2 suggestions).

hexylena commented on topics/assembly/images/denovo_assembly/galaxy_rerun.png:
this doesn't seem to be referenced in the text / we have `{% icon %}` for this, so you can remove this.
